packages/graph/hash_graph/bin/hash_graph/src/main.rs

```diff
@@ -15,8 +15,10 @@ impl fmt::Display for GraphError {
 
 #[tokio::main]
 async fn main() -> Result<(), GraphError> {
-    let connect_url = "postgres://postgres:postgres@localhost/postgres"; // TODO - stop hardcoding
-    let _datastore = PostgresDatabase::new(connect_url).await;
+    // TODO: stop hardcoding the connection parameters
+    let _datastore = PostgresDatabase::new("postgres", "postgres", "localhost", 5432, "graph")
```

Comment:
I have looked into that and it wasn't as straightforward as I have hoped, so I'd leave that to a follow-up.

Reply:
Cool, makes sense 👍

Reply:
@TimDiekmann I tried using [`dotenv`](https://crates.io/crates/dotenv) in the binary when I was testing out sqlx, does it not pick up the correct files, or are we not using that intentionally?

Reply:
The readme of `dotenv` states
> **Achtung!** This is a v0.* version! Expect bugs and issues all around. Submitting pull requests and issues is highly encouraged!

So I chose to not do that as a drive-by 🙃 
I guess this will not pick up `.env.local`.